com.unity.render-pipelines.core/Runtime/Utilities/ResourceReloader.cs

```diff
@@ -190,19 +190,23 @@ static UnityEngine.Object Load(string path, Type type, bool builtin)
 
             // Else the path is good. Attempt loading resource if AssetDatabase available.
             UnityEngine.Object result;
-            if (builtin && type == typeof(Shader))
+            if (builtin)
             {
-                result = Shader.Find(path);
+                if (type == typeof(Shader))
+                {
+                    result = Shader.Find(path);
+                }
+                else
+                {
+                    result = Resources.GetBuiltinResource(type, path);
+
+                    if (IsNull(result))
+                        result = AssetDatabase.GetBuiltinExtraResource(type, path);
+                }
             }
             else
             {
                 result = AssetDatabase.LoadAssetAtPath(path, type);
```

Comment:
```suggestion
      result = (type == typeof(Shader)) ? Shader.Find(path) : AssetDatabase.LoadAssetAtPath(path, type);
```

I wonder if we should use the `Shader.Find` here too.